src/adapters/supabase/helpers/log.ts

```diff
@@ -169,6 +232,13 @@ export class GitHubLogger implements Logger {
 
   warn(message: string | object, errorPayload?: string | object) {
     this.save(message, LogLevel.WARN, errorPayload);
+    this.sendDataWithJwt(message, errorPayload)
+      .then((response) => {
+        console.log("Log Notification Success:", response);
```

Comment:
shouldn't we use logger instead of `console.log` or `console.error`?

Reply:
Oh yea, I'd do so now